extension/src/pages.json

```diff
@@ -79,6 +79,15 @@
             "autoSync": {
                 "enabled": true
             }
+        },
+        {
+            "host": "emby*|:8096*",
```

Comment:
Is there a specific reason why `emby` would be in the host, or is that just a likely user preference?

Reply:
User preference. If they run emby under a custom domain I think it's likely that they will have emby.domain configured in dns. I personally do. 

Also, I think it's safe to assume most emby users are using the default 8096 for emby. The only case where they wouldn't be is if they are also running jellyfin (what emby was forked from) on the same server.